packages/mobile/src/screens/auth/components/header/header.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import {
+  LinearGradient,
+  Logo,
+  View,
+  HeaderButton,
+} from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+type Props = {
+  labelButton: string;
+  onPress: () => void;
+};
+
+const Header: FC<Props> = ({ labelButton, onPress }) => {
+  return (
+    <LinearGradient
+      start={{ x: 0, y: 0.4 }}
+      end={{ x: 0, y: 0.9 }}
+      colors={['#0C42A6', '#F4C50A']}
```

Comment:
Do we have these colors within ColorPalete? If not, let's add them.

Reply:
corrected